apps/backoffice-v2/src/pages/Statistics/components/PortfolioRiskStatistics/PortfolioRiskStatistics.tsx

```diff
@@ -205,16 +139,93 @@ export const PortfolioRiskStatistics: FunctionComponent<{
               </div>
             );
           })}
+          <div
+            key={'Report Statuses'}
+            className={'col-span-full min-h-[13.125rem] rounded-xl bg-[#F6F6F6] p-2'}
+          >
+            <Card className={'flex h-full flex-col px-3'}>
+              <CardHeader className={'pb-1'}>Report Status</CardHeader>
+              <CardContent>
+                <p className={'mb-8 text-slate-400'}>{'Merchant monitoring statuses'}</p>
```

Comment:
The pattern with the object is only for dynamic content or for using string templates.
```suggestion
                <p className={'mb-8 text-slate-400'}>Merchant monitoring statuses</p>
```